### PR TITLE
Small Issues fixed and Update all JSDoc

### PR DESCRIPTION
**Pull request by commcheck396**

### CODE REVIEW

**Review by SirKanaad26 — COMMENTED** [review]
JS Docs HTML Code - Do we need it?

**Comment by commcheck396** [comment]
> JS Docs HTML Code - Do we need it?

Idk, what do you think?

File: src/utils/seededRandom.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 let rand;
-
 /**
- * A hash function based on MurmurHash3's mixing function, developed by bryc
- * @param {string} str - The string to be hashed
- * @returns {() => number} - A function that returns a 32 bit hash value
+ * A transformation of the murmur3 hash function into a PRNG
```
Comment (Earth34r) [inline src/utils/seededRandom.js:3]: This isn't a PRNG, it's a hash function based on the mixing function of murmurhash3. Original description was a bit more accurate.

Comment (commcheck396) [reply src/utils/seededRandom.js:3]: Ok, will modify that later

**Review by Earth34r — APPROVED** [review]
Looks good to me, docs

**Pull request merged**